tests/acceptance/support/app/app.helpers.ts

```
@@ -0,0 +1,36 @@
+  const spawnOptions: SpawnOptionsWithoutStdio = {
+    shell: true,
+  };
+  const serverProcess = spawn("pnpm run start:prod", spawnOptions);
+
+  return serverProcess;
+}
+
+function killAppProcess(serverProcess: ChildProcessWithoutNullStreams | undefined): void {
+  if (serverProcess?.killed === false) {
+    serverProcess.kill("SIGTERM");
+    console.log("[acceptance-tests-api] Server process killed.");
```

ESLint

[error] Unexpected console statement.

(no-console)